site/content/3.10/arangograph/security-and-access-control/_index.md

```diff
@@ -14,40 +14,49 @@ access control:
 - Projects
 - Deployments
 
-For each of these resources you can perform various operations.
-For example, you can create a project in an organization and create a deployment inside a project.
+For each of these resources, you can perform various operations.
+For example, you can create a project in an organization and create a deployment
+inside a project.
 
 ## Locked resources
 
-In ArangoGraph, you can lock the resources to prevent accidental deletion. When a resource is locked,
-it cannot be deleted and must be unlocked first.
+In ArangoGraph, you can lock the resources to prevent accidental deletion. When
+a resource is locked, it cannot be deleted and must be unlocked first.
 
-The hierarchial structure of the resources (organization-project-deployment) is used in the locking functionality: if a child resource is locked
-(for example, a deployment), you cannot delete the parent project without unlocking that deployment first.
+The hierarchical structure of the resources (organization-project-deployment)
+is used in the locking functionality: if a child resource is locked
+(for example, a deployment), you cannot delete the parent project without
+unlocking that deployment first.
 
 {{< info >}}
-If you lock a backup policy of a deployment or an IP allowlist, CA certificate, and IAM provider of a project, it is still possible to delete
+If you lock a backup policy of a deployment or an IP allowlist, CA certificate,
+and IAM provider of a project, it is still possible to delete
 the corresponding parent resource without unlocking those properties first.
 {{< /info >}}
  
 ## Policy
 
-Various actions in ArangoGraph require different permissions, which can be granted to users via
-**roles**.
+Various actions in ArangoGraph require different permissions, which can be
+granted to users via **roles**.
 
 The association of a member with a role is called a **role binding**.
 All role bindings of a resource comprise a **policy**.
 
-Roles can be bound on an organization, project and deployment level (listed in the high to low level order, with lower
-levels inheriting permissions from their parents). This means that there is a unique policy per resource (an organization, a project or a deployment).
+Roles can be bound on an organization, project, and deployment level (listed in
+the high to low level order, with lower levels inheriting permissions from their
+parents). This means that there is a unique policy per resource (an organization,
+a project, or a deployment).
 
 For example, an organization has exactly one policy,
 which binds roles to members of the organization. These bindings are used to
 give the users permissions to perform operations in this organization.
+This is useful when, as an organization owner, you need to extend the permissions
+for an organization member. 
```

Comment:
Given that the current permissions structure is quite complicated (permissions, roles, role bindings, policies), I suggest putting the info about different permissions between org members and owners here, so that it's more visible.
Another suggestion is to point users here to the Policy page of an Organization on the AG dashboard, because it lists roles for both org owners and members.

Reply:
Done.